runtime/compiler/control/DLLMain.cpp

```diff
@@ -109,6 +109,7 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void * reserved)
             FIND_AND_CONSUME_ARG( EXACT_MATCH, "-Xhysteresis", 0);
             }
          FIND_AND_CONSUME_ARG( EXACT_MATCH, "-Xnoquickstart", 0); // deprecated
+         FIND_AND_CONSUME_ARG(EXACT_MATCH, "-XperfTool", 0);
```

Comment:
As per https://github.com/eclipse-openj9/openj9/issues/15795#issuecomment-1232061312 the option should be something like `-XX:+PerfTool` and `-XX:-PerfTool`. You should then only enable it if the `+` variant comes after the `-` variant.